pages/self-hosting/vercel.mdx

```diff
@@ -1,5 +1,7 @@
-import { Callout } from 'nextra-theme-docs'
+import { Callout } from "nextra-theme-docs";
+
+# Vercel as Frontend
 
 <Callout type="info" >
-  We're still working on a Vercel guide 
-</Callout>
\ No newline at end of file
+  Learnhouse is currently unsupported on Vercel.
```

Comment:
This may make people think that Learnhouse can't be run technically on Vercel, instead we should just let users know that a guide is being worked on